lib/genKeys.js

```diff
@@ -1,16 +1,11 @@
 'use strict'
 
-function genKeys (obj, set) {
+function genKeys (obj) {
   return Object.keys(obj)
-    .filter(notInSet, set)
     .filter(noObjects, obj)
     .map(keyIterator, obj)
 }
 
-function notInSet (key) {
-  return !this || this.has(key)
-}
-
```

Comment:
Ah balls!

Ok I'll add this back in and also a test to cover the path. None of the tests are reaching that code in istanbul btw